index.html

```diff
@@ -245,7 +245,46 @@ <h4 class="bolda" >About me</h4>
           </main>
         </section>
     
-    
+    <footer class="footer">
+      <h2 class="footer-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in <span class="touch">touch.</span></h2>
+      
+      <form  class="formx" action="https://formspree.io/f/xayagqrn" method="post">
+        <ul class="formul">
+        <li class="for"><label for="fullName"><input type="text" name="full-name" placeholder="  Full name" maxlength="30" id="fullName" required>
+        </label>
+      </li>
```

Comment:
[OPTIONAL] Please be careful with the indentation of your document. You can read more about [best practices for HTML & CSS here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)